docs/BoundedInts.md

```diff
@@ -0,0 +1,51 @@
+
+## The `BoundedInts` module {#sec-boundedints}
+
+The `Dafny.BoundedInts` module contains definitions of types and constants for fixed-bit-width integers.
+
+Dafny itself generally uses and reasons about mathematical, unbounded integers and natural numbers.
+However compiled programs, for efficiency of computation and storage, will generally use fixed-bit-width
+numbers. So Dafny defines them here, for use when needed.
```

Comment:
I would suggest that you use https://sembr.org/
This is because Markdown will compose the lines
into something readable by humans.
But when reading the written words,
the diffs get _much_ easier to see.